pkg/security/config/config.go

```diff
@@ -119,6 +119,22 @@ type Config struct {
 	// parameter empty to monitor all event types. If not already present, the `exec` event will automatically be added
 	// to this list.
 	ActivityDumpTracedEventTypes []model.EventType
+	// ActivityDumpBindRate defines the rate at which bind events can added to a dump
```

Comment:
actually i think the comments about the units of these values belong here

Reply:
Fixed on last commit, thank you !